srcjs/output_binding_image.js

```diff
@@ -90,94 +90,94 @@ $.extend(imageOutputBinding, {
       }
     }
 
-    if (!opts.coordmap)
-      opts.coordmap = [];
-
-    // When the image loads, initialize all the interaction handlers. When the
-    // value of src is set, the browser may not load the image immediately,
-    // even if it's a data URL. If we try to initialize this stuff
-    // immediately, it can cause problems because we use .naturalWidth and
-    // .naturalHeight, but they're available only after the image loads.
+    if (!opts.coordmap) {
```

Comment:
Need to test what happens if we have a plot without a valid coordmap, like a ggplot2 polar plot.